AppController/djinn.rb

```diff
@@ -5729,14 +5862,29 @@ def get_all_stats(secret)
           if app_name != "none"
             total_reqs, reqs_enqueued, collection_time = get_haproxy_stats(app_name)
             # Create the apps hash with useful information containing HAProxy stats.
-            all_stats["apps"][app_name] = {
-              "language" => @app_info_map[app_name]["language"].tr('^A-Za-z', ''),
-              "appservers" => @app_info_map[app_name]["appengine"].length,
-              "http" => @app_info_map[app_name]["nginx"],
-              "https" => @app_info_map[app_name]["nginx_https"],
-              "total_reqs" => total_reqs,
-              "reqs_enqueued" => reqs_enqueued
-            }
+            tries = 11
```

Comment:
I wanted something more than the usual number of tries which was 5 everywhere I believe, I will make that 10 and define a constant for it.